modules/ocf_mirrors/manifests/ftpsync.pp

```diff
@@ -1,20 +1,16 @@
 # ftpsync is a Debian tool for reliably mirroring Debian archives:
 # https://www.debian.org/mirror/ftpmirror
 define ocf_mirrors::ftpsync(
-    $cron_minute,
     $rsync_host,
+    $cron_minute = undef,
```

Comment:
Whoops, read the last part of your comment lol.

Yeah, probably true on that, we would want to sync periodically anyway, so I'll change this back, good point.